x/accounts/genesis.go

```diff
@@ -80,7 +80,6 @@ func (k Keeper) ImportState(ctx context.Context, genState *v1.GenesisState) erro
 }
 
 func (k Keeper) importAccount(ctx context.Context, acc *v1.GenesisAccount) error {
-	// TODO: maybe check if impl exists?
```

Comment:
Seems like a valid todo tbh, why is it being removed?

Reply:
I thought this TODO was about checking if v1.GenesisAccount is properly implemented, not whether it exists. I believed it's sufficient as each value is already being checked internally within importAccount().
Could I have misunderstood?

Reply:
I think the todo is about not adding an account that may not have an implementation after a hard fork.
Say your chain has a lockup account, but after you do a hard fork you forkot to add it, `InitGenesis` should panic.
This is a nice UX improvement and still valid. Basically, checking the account_type against the registered accounts in x/accounts.

Reply:
Thank you for the kind explanation. I'll close this PR and create a separate one. I'll close the issue as well.